examples/message/send_receive/SendReceive.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using NationalInstruments.SystemLink.Clients.Core;
+using NationalInstruments.SystemLink.Clients.Message;
+using Newtonsoft.Json;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.Message
+{
+    /// <summary>
+    /// Example for the SystemLink Message client API that opens two message
+    /// sessions, sends JSON messages from one, then reads those messages back
+    /// from the other.
+    /// </summary>
+    class SendReceive
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * See the configuration example for how a typical application
+             * might obtain a configuration.
+             */
+            var configuration = ExampleConfiguration.Obtain(args);
+
+            /*
+             * Open a session and subscribe to a couple of topics. Once
+             * subscribed, the server will queue messages published to any of
+             * these topics until read.
+             */
+            Console.WriteLine("Opening message read session...");
+
+            using (var readSession = MessageSession.Open(configuration))
+            {
+                readSession.Subscribe("example.data");
+                readSession.Subscribe("example.exit");
+                PublishMessages(configuration);
```

Comment:
Put a newline above this. It's not associated with the example subscription above, it's setup for the message reading example below. Perhaps it should even be moved below the "read session" comment?